client/main.lua

```diff
@@ -418,6 +431,21 @@ local function openVehicleSellMenu(targetVehicle)
     lib.showContext('vehicleMenu')
 end
 
+--- End test drive
+local function endTestDrive()
+    TriggerServerEvent('qbx_vehicleshop:server:deleteVehicle', testDriveVeh)
+    testDriveVeh = 0
+    inTestDrive = false
+    LocalPlayer.state:set('isInTestDrive', false, true)
```

Comment:
Do we need a local variable if we have a statebag set? Could we just use the statebag?